cloudhub-1/modules/ROOT/Edits/vpn-maintenance.adoc

```diff
@@ -0,0 +1,71 @@
+= Anypoint VPN Maintenance
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: runtime-manager::vpc-maintenance.adoc,
+
+
+== Anypoint VPN Update
+
+MuleSoft Anypoint VPN is a managed service that is underpinned by AWS Site-to-Site VPN connections. Using a VPN is important for any integrations or flows that require connectivity to your on-premises or other external platforms. Updating an Anypoint VPN that is not set up with at least a pair of tunnels that use BGP routing will cause downtime. See xref:vpn-high-availability.adoc[Anypoint VPN High Availability] for more information.
```

Comment:
I think you can remove "other" because it implies that the on-premises platform is external.